core/arch/arm/plat-rockchip/main.c

```diff
@@ -34,15 +35,20 @@ static const struct thread_handlers handlers = {
 
 void main_init_gic(void)
 {
-	vaddr_t gicc_base;
-	vaddr_t gicd_base;
+	vaddr_t gicc_base, gicd_base;
```

Comment:
Please keep one variable per line, that's our preferred style. And please initialize both to `0`.